web_app/calendar/helpers.py

```diff
@@ -132,6 +132,34 @@ def gen_day_cell(year, month, day):
     return full_day
 
 
+def gen_daily_cells():
+    tasks = get_daily_tasks()
+    tasks_list = ''
+
+    if tasks:
```

Comment:
You can use just _for_ without checking  _if tasks_
_for_ can handle empty list.